docs/_install-and-build/Installing-Geth.md

```diff
@@ -66,12 +60,6 @@ sudo apt-get install ethereum-unstable
 pkg install go-ethereum
 ```
 
-The `geth` command is then available on your system in `/usr/local/bin/geth`, you can start it on the testnet (for example) by using:
```

Comment:
It's nice to know what gets installed and where. Maybe it would make sense to list all the commands that become available once geth is installed (clef, puppeth, etc.)

Reply:
I agree! We should probably extend the document a bit to explain where the binary ends up and how to get the all other tools.